### DISCUSSION TITLE
Error after 2 second of live

### DISCUSSION THREAD

**Discussion by Adivor** [bluenviron/mediamtx#1107] [Questions]
Hello to all,
I send to my server debian my stream from OBS 8000 Kbps : rtmp://xxxxxxxx.com/rtsp/mystream
All ok, server shell show me " is publishing to path 'rtsp/mystream', 2 tracks (H264, MPEG4Audio)"

When i use rtmp://xxxxxxxx.com/rtsp/mystream in iphone app i receive signal video and audio but all stop work after 2 second (see screenshot). 

If i stream from smartphone with rtmp and OBS for receive from server all work but quality is very verly low and pixelated (see attached).

Server Load is low, dualcore VPS
SET readBufferCount: 1024
SET protocols: [tcp]

![2022-08-20 04_17_24-_var_www_www-root_data_www_mw2 it_rtsp_rtsp-simple-server yml - root@Peperoncini](https://user-images.githubusercontent.com/42887701/185725429-0e5f60c4-7cfa-47bd-bf46-4bba8f049834.png)
![IMG_1881](https://user-images.githubusercontent.com/42887701/185725433-a9a5ac49-4b8f-437e-a13f-d5b349aa4a71.PNG)

https://user-images.githubusercontent.com/42887701/185725971-6c915295-54f8-4072-b97e-db16a0048fcd.mp4

**Comment by aler9** [comment]
Hello, what are the Output settings of OBS Studio? what is the encoder?

[a comment by an automated account is not shown]
